nwb_conversion_tools/utils/specdatachunkiterator.py

```diff
@@ -0,0 +1,74 @@
+"""Authors: Cody Baker, Oliver Ruebel."""
+from typing import Iterable
+import numpy as np
+
+from hdmf.data_utils import AbstractDataChunkIterator, DataChunk
+
+
+class SpecDataChunkIterator(AbstractDataChunkIterator):
+    """DataChunkIterator that lets the user specify chunk shapes."""
+
+    def __init__(self, data: Iterable, full_shape: tuple, chunk_shape: tuple, dtype: str):
```

Comment:
Is there any way to include `super().__init__` here?

Reply:
I don't think ``super().__init__()`` will work here.``AbstractDataChunkIterator`` is an abstract class so calling ``super().__init__()`` should produce a ``TypeError: Can't instantiate abstract class ...`